arangod/Graph/BreadthFirstEnumerator.cpp

```diff
@@ -161,6 +161,12 @@ bool BreadthFirstEnumerator::next() {
           }
         }
 
+#ifdef USE_ENTERPRISE
+        if (!validDisjointPath(nextIdx, vId)) {
+          return;
+        }
+#endif
```

Comment:
This will then collide with our EE code (if we remove it)

Reply:
What I mean is: Either we can remove the `#ifdef` and `#endif` lines here https://github.com/arangodb/arangodb/blob/1ace9dc70e29921202085a5991b5f58b1e27a6c0/arangod/Graph/BreadthFirstEnumerator.cpp#L164, because there is an implementation in CE at https://github.com/arangodb/arangodb/blob/1ace9dc70e29921202085a5991b5f58b1e27a6c0/arangod/Graph/BreadthFirstEnumerator.cpp#L376.

Or the `#ifdef` stays, in which case `validDisjointPath` is not needed in CE at all, which means https://github.com/arangodb/arangodb/blob/1ace9dc70e29921202085a5991b5f58b1e27a6c0/arangod/Graph/BreadthFirstEnumerator.cpp#L376 - https://github.com/arangodb/arangodb/blob/1ace9dc70e29921202085a5991b5f58b1e27a6c0/arangod/Graph/BreadthFirstEnumerator.cpp#L381 can be completely removed.

Reply:
Understood, will fix it.